main_test.go

```diff
@@ -21,6 +22,17 @@ func TestReadBinary(t *testing.T) {
 	}
 }
 
+func TestReadBinary_IT(t *testing.T) {
```

Comment:
Can you add tests for a couple of italian words as well for some sanity checking?

Reply:
I made a test that is multilanguage, we just need to put some more examples in other languages.